deploy/kubernetes/metricbeat-kubernetes.yaml

```diff
@@ -14,12 +14,47 @@ data:
       # Reload module configs as they change:
       reload.enabled: false
 
-    # To enable hints based autodiscover uncomment this:
-    #metricbeat.autodiscover:
-    #  providers:
-    #    - type: kubernetes
-    #      node: ${NODE_NAME}
-    #      hints.enabled: true
+    metricbeat.autodiscover:
+      providers:
+        # To enable hints based autodiscover uncomment this:
+        #- type: kubernetes
+        #  node: ${NODE_NAME}
+        #  hints.enabled: true
+        # Uncomment the following to enable leader election provider that handles
+        # singleton instance configuration across the Daemonset Pods of the whole cluster
+        # in order to monitor some unique data sources, like kube-state-metrics.
+        # When enabling this remember to also delete the Deployment or just set the replicas of the
+        # Deployment to 0.
```

Comment:
This is too much wording IMO, if we do the docs right this should only be
```
# uncomment the following to enable collection from kube-state-metrics
```

Reply:
👍  I will try to improve it